build.gradle

```diff
@@ -6,130 +6,130 @@ def VS_2015_LIB_DIR = ""
 
 allprojects{
```

Comment:
Thanks for making the formatting changes! However, some of the indentation and brace placement is still off (see comments below).

For here and elsewhere, make sure there is a space before the brace.